pkg/importer/dataStream.go

```diff
@@ -266,6 +273,110 @@ func (d *DataStream) http() (io.ReadCloser, error) {
 	return countingReader, nil
 }
 
+type registryData struct {
+	dataDir string
+	file    *os.File
+}
+
+//_regData - global variable that stores temporary data of registy flow
+// to be deleted when teh flow ends
+var _regData = registryData{"", nil}
```

Comment:
Why can't this data belong to the datastream struct?

Reply:
This data is relevant only to import from registry flow

Reply:
I still think it would be preferable to have members (even if they are unused) in the datastream.  By using global variables, we are precluding ourselves from ever doing simultaneous registry imports in the same process.